### PR TITLE
Contribution page: Converting from "I" to "we" wording

### PR DESCRIPTION
**Pull request by spacegaier**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [X] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Comment by glmnet** [comment]
Actually this opinion can be removed from docs. It really is not adding technical documentation. 

Besides Otto changed his mind and there are some code automation tools which relies on rst so changing to markdown wouldn't be suggested. 

See also https://github.com/esphome/feature-requests/issues/803

File: guides/contributing.rst [hunk 1]
```diff
@@ -31,8 +31,8 @@ The ESPHome documentation is built using `sphinx <http://www.sphinx-doc.org/>`__
 Syntax
 ******
 
-In my opinion, Markdown would have been the much better choice in hindsight, but at the time
-I was setting up the documentation good Doxygen integration was key to me. Anyway, here's a quick
+In hindsight, Markdown would have been the much better choice, but at the time we were
+setting up the documentation good Doxygen integration was key to us. Anyway, here's a quick
 RST primer:
```
Comment (frenck) [inline guides/contributing.rst]:
```suggestion
Here's a quick RST primer:
```

**Review by frenck — CHANGES_REQUESTED** [review]
@spacegaier How about just this :)

**Comment by spacegaier** [comment]
Looks like someone else already changed that (https://github.com/esphome/esphome-docs/pull/1052) => my PR is now obsolete.

**Pull request closed without merging**